pallets/pool-system/src/lib.rs

```diff
@@ -12,7 +12,6 @@
 // GNU General Public License for more details.
 #![cfg_attr(not(feature = "std"), no_std)]
 #![allow(clippy::or_fun_call)]
-#![feature(thread_local)]
```

Comment:
Comment: Approved by @mustermeiszer as this is not necessary and blocked WASM building CI.